rails_app/app/jobs/deals_scrape_job.rb

```diff
@@ -0,0 +1,35 @@
+class DealsScrapeJob < ApplicationJob
+  queue_as :default
+
+  @@graphite = GraphiteAdapter.new
+
+  def perform(*args)
+    # Do something later
+    start_time = Time.now
+    deals_to_parse.each do |deal|
+      # new_deal_status = Settings.parser_helpers.join_parsers[deal.provider].constantize.new(deal.url)
+      if deal.updated_at.nil? or deal.updated_at + deal.scrape_interval < Time.now
```

Comment:
updatet_at wont be nil coz at record create its set

Reply:
ok, need to delete it then